Fix the api/client/ps unit tests.

vdemeester opened the pull request:
TestFormat was depending on the time so 😱 (comparing again 1970-01-01).. at some point it was bounded to fail >_<. Updating it to be _not_ time-dependent :). Fixing my own mistakes 🐷.

🐸

PS: all unit tests breaks on all build from now on until this fixes it :sweat_smile:.

Signed-off-by: Vincent Demeester vincent@sbr.pm

ripcurld00d commented:
LGTM :+1:

thaJeztah commented on api/client/ps/formatter_test.go:
```diff
@@ -33,8 +38,8 @@ func TestFormat(t *testing.T) {
-containerID1        ubuntu              ""                  45 years ago                                                foobar_baz
-containerID2        ubuntu              ""                  45 years ago                                                foobar_bar
+containerID1        ubuntu              ""                  24 hours ago                                                foobar_baz
+containerID2        ubuntu              ""                  24 hours ago                                                foobar_bar
```
Is this going to be flaky (23 hours <--> 24 hours?). Would it be more stable to use 36 hours in the past, so that it would output e.g. `1 day ago`

thaJeztah replied:
(not sure, just wondering)

runcom replied:
make sense

vdemeester committed:
Fix the api/client/ps unit tests.

TestFormat was depending on the time so (comparing again
1970-01-01).. at some point it was bounded to fail >_<. Updating it to
be *not* time-dependent :).

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

runcom commented:
LGTM

thaJeztah commented:
LGTM (if tests are green :smile:)

Thanks @vdemeester!

thaJeztah commented:
Windows failed, but looks to be a flaky test: https://github.com/docker/docker/pull/18613 (see https://github.com/docker/docker/pull/18613#issuecomment-166298416)

thaJeztah commented:
Merging as the failure should not be related
